firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -199,8 +199,9 @@ class ActivityStreamTest: BaseTestCase {
         navigator.nowAt(NewTabScreen)
         // Open one of the sites from Topsites and wait until page is loaded
         // Long tap on apple top site, second cell
-        waitForExistence(app.collectionViews.cells.element(boundBy: 4), timeout: 3)
-        app.collectionViews.cells.element(boundBy: 4).press(forDuration: 1)
+        waitForExistence(app.collectionViews["FxCollectionView"].cells
```

Comment:
If the test works on iPhone, wondering if we should use this on if iPad or this will be as consistent as it is now for iPhone too..